Tests/Transformers/MysqlToRequestTest.php

```diff
@@ -226,7 +240,32 @@ public function updateAll() {
             'payload'     => json_encode(['name' => 'myValue'])
         ]);
 
-        $this->assertEquals($expected, $this->mysqlToRequest->transform($query));
+        $this->assertEquals($expected, $this->createMysqlToRequest()->transform($query));
+    }
+
+    /**
+     * @test
+     * @group  unit
+     * @covers ::__construct
+     * @covers ::transform
+     * @covers ::<private>
+     */
+    public function updatePatch() {
```

Comment:
I thought about this but wanted to ask whether it would be smarter to increase the limit for all test files via the config file. I'm not sure whether that's possible though.